pxtblocks/blocklyloader.ts

```diff
@@ -585,7 +585,19 @@ namespace pxt.blocks {
     }
 
     function initField(i: any, ni: number, fn: pxtc.SymbolInfo, ns: pxtc.SymbolInfo, pre: string, right?: boolean, type?: string, nsinfo?: pxtc.SymbolInfo): any {
-        if (pre)
+        if (pre && pre.indexOf('`') > -1) {
+            // parse and create icon fields for every inline icon
+            let regex = /([^`]+|(`([^`]*)`))/gi;
```

Comment:
what are you trying to match here? https://rise4fun.com/Rex/NY0

Reply:
text without a backticks, and text within a pair of backticks

Reply:
I see that but I don't get the logic why you match text without backtick. Can we simplify this (and document it?)

Reply:
Sure

Reply:
You use + in between the backticks